src/collections/_documentation/performance/distributed-tracing.md

```diff
@@ -12,105 +12,325 @@ Sentry's Performance features are currently in beta. For more details about acce
 
-View transaction events by clicking on the "Transactions" pre-built query in [Discover]({%- link _documentation/performance/discover/index.md -%}), or by using a search condition `event.type:transaction` in a [Discover Query Builder]({%- link _documentation/performance/discover/query-builder.md -%}) view.
+**Nested Spans**
 
-When you open a transaction event in Discover, you'll see the **span view** at the top of the page. Other information the SDK captured as part of the Transaction event, such as breadcrumbs, will also be displayed.
+Though our examples above had four levels in their hierarchy (trace, transaction, span, child span) there's no set limit to how deep the nesting of spans can go. There are, however, practical limits: transaction payloads sent to Sentry have a maximum allowed size, and as with any kind of logging, there's a balance to be struck between your data's granularity and its usability. 
 
-[{% asset performance/discover-span.png alt="Discover span showing the map of the transactions (aka minimap) and the black dashed handlebars for adjusting the window selection." %}]({% asset performance/discover-span.png @path %})
+**Zero-duration Spans**
+
+It's possible for a span to have equal start and end times, and therefore be recorded as taking no time. This can occur either because the span is being used as a marker (such as is done in [the browser's Performance API](https://developer.mozilla.org/en-US/docs/Web/API/Performance/mark)) or because the amount of time the operation took is less than the measurement resolution (which will vary by service).
+
+**Clock Skew**
+
+If you are collecting transactions from multiple machines, you will likely encounter **clock skew**, where timestamps in one transaction don't align with timestamps in another. For example, if your backend makes a database call, the backend transaction logically should start before the database transaction does. But if the system time on each machine (those hosting your backend and database, respectively) isn't synced to common standard, it's possible that won't be the case. It's also possible for the ordering to be correct, but for the two recorded timeframes to not line up in a way that accurately reflects what actually happened.
+
+Because there is no way for Sentry to judge either the relative or absolute accuracy of your timestamps, it does not attempt to correct for this. And while you can reduce clock skew by using Network Time Protocol (NTP) or your cloud provider's clock synchronization services, you may still notice small discrepancies in your data, as synchronizing clocks on small intervals is challenging.
+
+**What We Send**
+
+Individual spans aren't sent to Sentry. Rather, those spans are attached to their containing transaction, and the transaction is sent as one unit. This means that no span data will be recorded by Sentry's servers until the transaction to which they belong is closed and dispatched. 
```

Comment:
Isn't it the case, though, that a span which isn't part of a transaction (in other words, doesn't have `transaction` set and doesn't have an ancestor with `transaction` sent) gets dropped? ([Here](https://github.com/getsentry/sentry-javascript/blob/cf2bc7e1264f1b5a74e6e9dae4d4ebf5367632ea/packages/apm/src/span.ts#L287-L290) is where that happens, I believe.) So I feel like it is important, so that those manually instrumenting their code know that they have to create a transaction in order for their stuff to get recorded, no?

And you may be right - it may be temporary, though I was under the impression that sending individual spans involved enough overhead that we didn't ever want to do it. I could easily be wrong, though. @HazAT - care to weigh in?

To your theoretical example, I'd argue that in that case we're still only sending transactions, because once you break it down like that, each span becomes its own transaction, which would radically change what you see in the UI.

Reply:
> So I feel like it is important, so that those manually instrumenting their code know that they have to create a transaction in order for their stuff to get recorded, no?

Yes, that's why we say "the transaction is sent as one unit"; "no span data will be recorded by Sentry's servers until the transaction to which they belong is closed and dispatched".

"Individual spans aren't sent to Sentry." is true today, but felt to me redundant and over specific, that's why I suggested the edit.

Reply:
I think I meant it as a little bit of a warning/debugging hint, also, like, 'Hey, wondering why your spans aren't showing up in Sentry? This is why."

I tightened up the wording a bit, though, so hopefully it will read as less redundant.